tests/loading/schema/paths/operation/responses/schema/test_reference_validation.py

```diff
@@ -10,6 +12,9 @@
 )
 
 
+DIR = os.path.join(os.path.dirname(os.path.abspath(__file__)), '../../../../../')
```

Comment:
An *easier* to read way to do this is to do the following.

```python
import flex
BASE_DIR = os.path.dirname(os.path.dirname(flex.__file__))
```

This will give you the root folder of the project.